assets/openshift-state-metrics/deployment.yaml

```diff
@@ -61,7 +61,7 @@ spec:
         - --port=8081
         - --telemetry-host=127.0.0.1
         - --telemetry-port=8082
-        image: quay.io/openshift/origin-openshift-state-metrics:4.2
+        image: quay.io/openshift/origin-openshift-state-metrics:latest
```

Comment:
This should be pinned no?

Reply:
IMHO doesn't really matter for this component :shrug: